### DISCUSSION TITLE
The tutorial seems to be broken?

### DISCUSSION THREAD

**Discussion by jgoodman75** [go-spatial/tegola#862] [Q&A] [answered]
I noticed some of the links in the tutorial return a 404 error. For example https://openlayers.org/en/v4.5.0/build/ol.js. On the other hand, changing 4.5.0 to 4.6.5 works. Also noticed that https://tegola-osm-worker.go-spatial.org gets a 404.

Forgive me if I am mistaken. I am really interested in this project!

**Comment by ARolek** [comment, marked as answer]
@jgoodman75 the docs are pretty out of date. We need to find some time to get them updated. I will look into these particular issues to see if we can get them addressed.
